include/nbl/builtin/hlsl/spirv_intrinsics/core.hlsl

```diff
@@ -153,6 +153,11 @@ template<class T, class U>
 [[vk::ext_instruction(spv::OpBitcast)]]
 T bitcast(U);
 
+template<class T> // DXC Workaround issue TODO https://github.com/microsoft/DirectXShaderCompiler/issues/6576
+[[vk::ext_capability(spv::CapabilityPhysicalStorageBufferAddresses)]]
+[[vk::ext_instruction(spv::OpBitcast)]]
+T bitcast(uint64_t);
```

Comment:
no this will will make `uint64_t` to `double` bitcasts emit the capability.

You need to make a `spirv::is_physical_storage_ptr<T>` trait that we can use to detect either U or T being a spirv pointer type

Reply:
Done

Reply:
doesn't look done, this will match `T=double`

Reply:
did you mean to remove?

Reply:
Yes, I don't think we're using it anymore.